### PR TITLE
Update annotations deps to 1.7

### PR DESCRIPTION
**Pull request by reidbaker**
- Update annotation dependency
- Add changelog for update

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/camera/camera_android/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.10.8+9
+
+* Update annotations lib to 1.7.0.
```
Comment (gmackall) [inline packages/camera/camera_android/CHANGELOG.md:3]: Here and in other places, I believe this should be "Updates..."

https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

**Comment by bparrishMines** [comment]
Sorry, I just approved the dependabot update for `webview_flutter_android`: https://github.com/flutter/packages/pull/4891. You can skip that package. Should I leave future updates for this dep to the Android team?

**Comment by reidbaker** [comment]
> Sorry, I just approved the dependabot update for `webview_flutter_android`: #4891. You can skip that package. Should I leave future updates for this dep to the Android team?

No it is totally fine. I noticed that I got 1-2 and decided to just update the whole repo.

File: packages/camera/camera_android/CHANGELOG.md (hunk 1 above)
Comment (reidbaker) [reply packages/camera/camera_android/CHANGELOG.md]: Yeah, thanks this always bites me. I need to write a pre commit script for this.

**Comment by reidbaker** [comment]
> > Sorry, I just approved the dependabot update for `webview_flutter_android`: #4891. You can skip that package. Should I leave future updates for this dep to the Android team?
> 
> No it is totally fine. I noticed that I got 1-2 and decided to just update the whole repo.

Actually I got 40% of the way through modifying the update dependencies command to make this super easy in the future but I wanted to get this pr up the same day and reverted those changes locally.

**Review by gmackall — APPROVED** [review]

**Comment by stuartmorgan** [comment]
Is this ready to land?

**Comment by reidbaker** [comment]
it was but now has conflicts. Will update and try to land this week.

**Pull request merged**